### ISSUE TITLE
Bug Report: Price Updating Fails for Multiple Systems with Structures

### ISSUE THREAD

**Issue by MikFa** [EVEIPH/EVE-IPH#187]
**Problem Description:**
When using price profiles targeting multiple systems, especially those with structures, the price update functionality does not work as expected. The current implementation only updates prices for a single system, which is the last one assigned in the SentItems list.

**Details:**

- The UpdateStructureMarketOrders function is called with the PriceSystem parameter set to the SystemID of the last item in SentItems (see frmMain.vb, line 8976-9006).
```
 If SentItems(i).SystemID <> "" Then
                    DBCommand = New SQLiteCommand("SELECT regionID FROM SOLAR_SYSTEMS WHERE solarsystemID = '" & SentItems(i).SystemID & "'", EVEDB.DBREf)
                    readerPrices = DBCommand.ExecuteReader
                    readerPrices.Read()
                    RegionID = CStr(readerPrices.GetInt64(0))
                    readerPrices.Close()
                    DBCommand = Nothing
                    PriceSystem = SentItems(i).SystemID
                Else
                ....
                 If Not ESIData.UpdateStructureMarketOrders(PriceRegions, PriceSystem, SelectedCharacter.CharacterTokenData, pnlProgressBar) Then
```
- This function does not handle multiple PriceSystem values, causing it to ignore all systems except the one assigned last.
- Additionally, if the last SystemID in SentItems corresponds to Jita, the function fails to find any structures because Jita does not contain player-owned structures (see ESI.vb, starting at line 1146).
```
 If PriceSystem <> "" Then
            ' Only look up structures in this system
            SQL = "SELECT DISTINCT STATION_ID FROM STATIONS WHERE SOLAR_SYSTEM_ID =" & PriceSystem & " And STATION_ID >70000000"
        Else
```
**Suggested fix:**
Refactor UpdateStructureMarketOrders to handle multiple systems or remove it entirely. 

1. Instead of assigning a single SystemID to PriceSystem, it should accept a list of SystemIDs and iterate through them for price updates.
2. Set pricesystems to empty string for the call. 

**Impact:**
This bug prevents users from accurately updating prices for multiple systems, especially those with player-owned structures, and may limit the usability of the tool for players operating in various systems.

**Next Steps:**
I plan to propose a solution work on a pull request to address this issue if it is okay. However, due to time constraints, I will not be able to work on this until Wednesday or Thursday this week. I am submitting this issue report to document the problem.

**Comment by Zifrian** [comment]
Many thanks for the debugging, this has been driving me crazy for the past few weeks. I'm planning on a patch soon (hopefully this week) for the expansion so I can do it then.

**Comment by MikFa** [comment]
You're welcome, I'm glad to hear that my analysis was helpful. I apologize for not being able to contribute code myself due to time constraints this past week. I'm looking forward to the upcoming patch. Thank you for addressing this issue.

**Comment by Zifrian** [comment]
I put a fix in for this in Build 5.0.8735.42602 just now. Let me know if it works correctly for you.
